src/overlays/actors/ovl_Oceff_Wipe/z_oceff_wipe.c

```diff
@@ -111,7 +111,7 @@ void OceffWipe_Draw(Actor* thisx, GlobalContext* globalCtx) {
     gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(globalCtx->state.gfxCtx, "../z_oceff_wipe.c", 375),
               G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
 
-    if (this->actor.params) {
+    if (this->actor.params) { //Params nonzero == Song of Time
```

Comment:
id say instead of the comment we can just do `if (params != OCEFF_WIPE_ZELDAS)`
this probably shouldnt have been written as a boolean comparison anyway, but it was an error during decopmilation. we can change it now

Reply:
Additionally, you can remove the underscrore in the name for both enums
`OceffWipe_Type` -> `OceffWipeType`

then this is good to go